src/Tasks/Microsoft.NET.Build.Tasks/ResolvePackageAssets.cs

```diff
@@ -1520,7 +1520,8 @@ string GetPackageDiagnosticLevel(LockFileLibrary package)
                         log.LibraryId == package.Name &&
                         log.TargetGraphs.Any(tg =>
                         {
-                            var parsedTargetGraph = NuGetFramework.Parse(tg);
+                            var parts = tg.Split(Path.AltDirectorySeparatorChar);
```

Comment:
Maybe use LockFile.DirectorySeparatorChar directly, since that's what NuGet uses.

Reply:
Good point. Didn't check if that was actually available to use.